src/main/pages/che-7/installation-guide/examples/system-variables.adoc

```diff
@@ -1,4 +1,4 @@
-[id="che-server"]
+[id="LChe-server"]
```

Comment:
No idea if this commit should be dropped, I was following the contributing guide and had some changes to the repo.